makefile

```diff
@@ -0,0 +1,14 @@
+.PHONY: format-check
```

Comment:
the `Makefile` is typically capitalized by convention, unless there is a reason to use `makefile`

Reply:
yees 👍